layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi_4.19.bbappend

```diff
@@ -13,6 +13,7 @@ SRC_URI_append = " \
 	file://0001-Add-npe-x500-m3-overlay.patch \
 	file://0006-overlays-Add-Hyperpixel4-overlays.patch \
 	file://0001-waveshare-sim7600-Add-dtbo-for-this-modem.patch \
+    file://0001-seeed-studio-can-bus-v2-Add-dtbo-for-this-can-bus.patch \
```

Comment:
Hello @acostach, I try to run the build.
I cloned `balena-yocto-scripts` inside `balena-raspberry`. When I execute your command I get
```
$ balena-yocto-scripts/build/barys -b build_pi3 -m raspberrypi3 --shared-sstate /home/nikospetridis/sstate-cache/ --shared-downloads /home/nikospetridis/downloads/ -d
Building JSON manifest...
audited 3 packages in 1.478s
found 0 vulnerabilities

...Done
readlink: illegal option -- f
usage: readlink [-n] [file ...]
/Users/nikospetridis/projects/temp/balena-raspberrypi/balena-yocto-scripts/build/../../layers/poky/oe-init-build-env: line 45: /scripts/oe-buildenv-internal: No such file or directory
sed: 1: "conf/local.conf": command c expects \ followed by text
sed: 1: "conf/local.conf": command c expects \ followed by text
sed: 1: "conf/local.conf": command c expects \ followed by text
[000000006][LOG]BalenaOS build initialized in directory: build_pi3.
[000000006][LOG]Run build for raspberrypi3: MACHINE=raspberrypi3 bitbake resin-image
[000000006][LOG]This might take a while ...
env: bitbake: No such file or directory
[000000006][LOG]Build for raspberrypi3 failed. Check failed log in build_pi3/tmp/log/cooker/raspberrypi3 .
[000000006][LOG]If build for raspberrypi3 succeeded, final image should have been generated here:
[000000006][LOG]   build_pi3/tmp/deploy/images/raspberrypi3/resin-image-raspberrypi3.resinos-img
[000000006][LOG]Done.
```
I have copied `layers/meta-balena-raspberrypi/conf/samples/local.conf.sample` and renamed it to local.conf, tried to put it in several directories (inside `conf/` directory) but still got the same error when tried to build.
Am I missing something, can you please advise?
Thank you in advance